rasa/utils/tensorflow/layers.py

```diff
@@ -2,9 +2,12 @@
 from typing import List, Optional, Text, Tuple, Callable, Union, Any
 import tensorflow as tf
 import tensorflow_addons as tfa
+
+# TODO: The following is not (yet) available via tf.keras
+from keras.utils.control_flow_util import smart_cond
```

Comment:
But I see that `keras` is removed as a dependency in the poetry file. Does it come with the tensorflow package anyway?

Reply:
yep - https://github.com/keras-team/keras#installation

Reply:
Hahah, and then there are modules that are not in `tf.keras` 🤦